variables.tf

```diff
@@ -28,4 +28,29 @@ variable "endpoint_type" {
   }
 }
 
+variable "create_access_group" {
+  type        = bool
+  description = "Whether to create an access group for the secrets group."
+  default     = true
```

Comment:
Perhaps default this false in the module (otherwise user will always have to enter a value for `access_group_roles`